README.md

```diff
@@ -144,13 +144,24 @@ async fn async_main() {
 }
 ```
 
-The `Cargo.toml` will contain the following dependencies:
+### Simplified writing
 
-```toml
-[dependencies]
-nominal-api = "0.867.0"
-nominal-streaming = "0.2.0"
-tokio = { version = "1", features = ["full", "tracing"] }
+As a more direct shorthand for the above, you can use a writer:
+
+```rust
+let stream = NominalDatasetStream::new_with_consumer(
+    core_consumer(),
+    NominalStreamOpts::default()
+);
+let cd = ChannelDescriptor::new("channel_1");
```

Comment:
can we make this & the other examples use the builder, and only show the "new_with_consumer" function in an "advanced use" section at the bottom?